src/apps/project-home/ProjectHome.tsx

```diff
@@ -64,6 +69,12 @@ export const ProjectHome = (props: ProjectHomeProps) => {
     }
   }, [isAdmin]);
 
+  useEffect(() => {
```

Comment:
Since `ProjectHome` gets only rendered once, with the initial props, I guess you could skip the effect, and just initialize the state from the props?

I.e.

```ts
const [availableLayers, setAvailableLayers] = 
  useState(props.availableLayers);

// ...

/*
useEffect(() => {
  if (props.availableLayers) {
    setAvailableLayers(props.availableLayers);
  }
}, [props.availableLayers]);
*/
```